tasks/patches/add-sbom-and-push.yaml

```diff
@@ -119,6 +119,24 @@
       name: varlibcontainers
     workingDir: $(workspaces.source.path)
 
+- op: add
+  path: /spec/steps/-
+  value:
+    image: $(params.BUILDER_IMAGE)
+    name: expose-base-images-digests
+    resources: {}
+    script: |
+      base_images=()
+      for base_image in $(grep "^FROM " $1 | cut -d" " -f2); do
```

Comment:
whati si `$1` ?

I would just print all image which were used -> `buildah images --format '{{ .Name }}:{{ .Tag }}@{{ .Digest }}' >$(results.BASE_IMAGES_DIGESTS.path)`. And I would put it at the beginning of `inject-sbom-and-push` step, to reduce number of steps (due to quota limits.)